### PR TITLE
[docs] Update issue and discussion templates

### PR DESCRIPTION
**Pull request by amyegan**
Updates the existing [issue template](https://github.com/vercel/vercel/issues/new/choose) and adds [new discussion forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms) to direct Vercel platform questions to https://github.com/vercel/community/discussions

### CODE REVIEW

**Review by cb1kenobi — DISMISSED** [review]

File: .github/DISCUSSION_TEMPLATE/ideas.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+body:
+- type: markdown
+  attributes:
+    value: |
+      > Note: For ideas not related to the CLI or API, please go to the [Vercel Community](https://github.com/orgs/vercel/discussions/categories/ideas)
+- type: textarea
+  attributes:
+    label: Idea
+  validations:
+    required: true
```
Comment (styfle) [inline .github/DISCUSSION_TEMPLATE/ideas.yml]: I see you only added [3 of the 6 types](https://github.com/vercel/vercel/discussions/new/choose) as templates.

Should we disable all of these types so there is only one button?

Or should we add all 6 types as templates?

image

Comment (amyegan) [reply .github/DISCUSSION_TEMPLATE/ideas.yml]: I picked the ones that were most commonly used, but I can add it for them all if that seems more appropriate

Comment (styfle) [reply .github/DISCUSSION_TEMPLATE/ideas.yml]: I think it needs to be for all or else it won't solve the problem

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -4,8 +4,8 @@ contact_links:
     url: https://vercel.com/support/request
     about: Report a bug using the Vercel support form
   - name: Feature Request
-    url: https://github.com/vercel/vercel/discussions/new?category=ideas
+    url: https://github.com/vercel/community/discussions/new?category=ideas
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://github.com/orgs/vercel/discussions/new?category=ideas
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -4,8 +4,8 @@ contact_links:
...
     about: Share ideas for new features
   - name: Ask a Question
-    url: https://github.com/vercel/vercel/discussions/new?category=help
+    url: https://github.com/vercel/community/discussions/new?category=help
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://github.com/orgs/vercel/discussions/new?category=help
```

File: .github/DISCUSSION_TEMPLATE/ideas.yml (hunk 1 above)
Comment (amyegan) [reply .github/DISCUSSION_TEMPLATE/ideas.yml]: Done 😄

**Comment by styfle** [comment]
@amyegan Please fix the lint errors. You can run `pnpm prettier-check --write` to fix it

**Review by styfle — APPROVED** [review]

**Pull request merged**